### ISSUE TITLE
Avout javax.inject dependency

### ISSUE THREAD

**Issue by decebals** [avaje/avaje-inject#166]
Hi,

I noticed that the `javax.inject` depndency with `jakarta.inject`. 
Do you intend to maintain (add new features, support, ..) both variants in parallel?

**Comment by rbygrave** [comment]
We are already supporting both yes. 6.x for Jakarta.inject and 5.x for
javax.inject

Mentioned in the docs
https://avaje.io/inject/

There is a branch for the javax code and yes we are maintaining both
versions going forward.  This means all changes are made as PRs that are
merged into both branches.

**Comment by rbygrave** [comment]
I think I can close this now.

TLDR: 
- for javax.inject use version 5.x (latest now 5.16) 
- for jakarta.inject use version 6.x (latest now 6.16)

We are maintaining both branches. For example, the fix for #167 is now in version 5.15 and version 6.16. 
Re-open or add a further comment if that isn't clear.

Cheers, Rob.

**Comment by decebals** [comment]
> Mentioned in the docs
https://avaje.io/inject/

I read that. I ask you because I see that the versions [5.14](https://mvnrepository.com/artifact/io.avaje/avaje-inject/5.14) and [5.15](https://mvnrepository.com/artifact/io.avaje/avaje-inject/5.15) published in Maven Central Repository have `jakarta.inject` as dependency instead of `javax.inject` and the last version with `javax.inject` as dependency is [5.13](https://mvnrepository.com/artifact/io.avaje/avaje-inject/5.13).
Am I wrong with something?

**Comment by rbygrave** [comment]
Oh yeah, I see the confusion. So this dependency below looks like a `jakarta.inject` one but is actually still `javax.inject`.

```xml
<dependency>
  <groupId>jakarta.inject</groupId>
  <artifactId>jakarta.inject-api</artifactId>
  <version>1.0.5</version>
</dependency>
```

Now the reason for that is because the old javax.inject dependency didn't include a `module-info`.  So this dependency is a updated / newer version of javax.inject but with module-info.

The javax.inject artifacts are no longer allowed to be released into the javax namespace - hence it is released as `jakarata.inject : jakarta.inject-api : 1.0.5`.

So for example, downloading the jar from https://repo1.maven.org/maven2/jakarta/inject/jakarta.inject-api/1.0.5/jakarta.inject-api-1.0.5.jar ... and open up the jar using zip etc we will see it holds javax.inject code + META-INF/versions/9/module-info.class

In summary, the 5.x releases are still `javax.inject` and that dependency is confusing but correct in the sense it is the latest one for `javax.inject`.

**Comment by decebals** [comment]
Thanks for the explanations! I understand now.
